modules/ephemeral-storage-sharing-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-sharing-configmaps-across-namespaces_{context}"]
+= Sharing a config map across namespaces
+
+[role="_abstract"]
+To share a config map across namespaces in a cluster, you create a `SharedConfigMap` custom resource (CR) instance for that config map.
+
+.Prerequisites
+
+You must have permission to perform the following actions:
+
+* Create instances of the `sharedconfigmaps.sharedresource.openshift.io` custom resource definition (CRD) at a cluster-scoped level.
+* Manage roles and role bindings across the namespaces in the cluster to control which users can get, list, and watch those instances.
+* Manage roles and role bindings across the namespaces in the cluster to control which service accounts in pods that mount your Container Storage Interface (CSI) volume can use those instances.
+* Access the namespaces that contain the Secrets you want to share.
+
+.Procedure
+
+. Create a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: sharedresource.openshift.io/v1alpha1
+kind: SharedConfigMap
+metadata:
+  name: my-share
+spec:
+  configMapRef:
+    name: <name of configmap>
+    namespace: <namespace of configmap>
+EOF
+----
```

Comment:
Missing content, most of which is similar to the procedure for accessing RHEL entitlements in a build:

- Create a `ClusterRole` for the `SharedConfigMap`
- Create a `Role` and `RoleBinding` for the Shared Resource CSI Driver itself

At this point the ConfigMap is now ready to be shared across namespaces.